includes/class-wc-product-variable.php

```diff
@@ -183,6 +135,45 @@ public function get_price_html( $price = '' ) {
 	}
 
 	/**
+	 * Return a products child ids.
+	 *
+	 * @return array Children ids
+	 */
+	public function get_children( $context = 'view' ) {
+		if ( is_bool( $context ) ) {
+			wc_deprecated_argument( 'visible_only', '2.7', 'WC_Product_Variable::get_visible_children' );
+			return $context ? $this->get_visible_children() : $this->get_children();
+		}
+		if ( has_filter( 'woocommerce_get_children' ) ) {
+			wc_deprecated_function( 'The woocommerce_get_children filter', '', 'woocommerce_product_get_children or woocommerce_product_get_visible_children' );
+		}
+		return apply_filters( 'woocommerce_get_children', $this->children, $this, false );
+	}
+
+	/**
+	 * Return a products child ids - visible only.
+	 *
+	 * @since 2.7.0
+	 * @return array Children ids
+	 */
+	public function get_visible_children( $context = 'view' ) {
+		if ( has_filter( 'woocommerce_get_children' ) ) {
+			wc_deprecated_function( 'The woocommerce_get_children filter', '', 'woocommerce_product_get_children or woocommerce_product_get_visible_children' );
+		}
+		return apply_filters( 'woocommerce_get_children', $this->visible_children, $this, true );
+	}
+
+	/**
+	 * Return an array of attributes used for variations, as well as their possible values.
+	 *
+	 * @param  string $context
```

Comment:
Comment can be removed since there is no $context attribute on `get_variation_attributes`.